.github/workflows/acceptance-tests.yml

```diff
@@ -0,0 +1,60 @@
+name: 'Acceptance Tests'
+
+on:
+  workflow_dispatch: {}
+  workflow_call: 
+    inputs:
+      parent-event-name: # we want to run all the acceptance tests when the worflow is run in master (push)
+        required: true
+        type: string
```

Comment:
[q] Is master the full 12 hour run on every merge? I wonder instead do we only want to run the concurrent tests on master automatically, with an on-demand run for acceptance tests/on feature branches?

Reply:
Looking at the filters, we only run the cluster sim tests here? IF that is true, can you update the comment to indicate it's a partial run?

Reply:
> Looking at the filters, we only run the cluster sim tests here? 
yes, the workflow runs only 1 TF resource which takes ~24 minutes to complete. 

> IF that is true, can you update the comment to indicate it's a partial run?
sure

Reply:
updated